projects/angular/src/data/tree-view/tree-node.ts

```diff
@@ -188,6 +199,20 @@ export class ClrTreeNode<T> implements OnInit, OnDestroy {
     this.subscriptions.push(
       this._model.loading$.pipe(debounceTime(0)).subscribe(isLoading => (this.isModelLoading = isLoading))
     );
+
+    this.subscriptions.push(
+      this.typeAheadKeyEvent.pipe(debounceTime(TREE_TYPE_AHEAD_TIMEOUT)).subscribe((bufferedKeys: string) => {
+        this.focusManager.focusNodeStartsWith(bufferedKeys, this._model);
```

Comment:
does this match strictly on case? So if your text is "Eight" you won't match it with "eight"?

Edit: from the tests below it looks like this matches case insensitive just fine.

Reply:
Right, it's case insensitive.